scripts/delete_packer_vm.py

```
@@ -0,0 +1,64 @@
+        print(f"Error connecting to Proxmox: {e}")
+        return
+
+    vm_id = 500
+
+    try:
+        # Check if the VM exists
+        proxmox.nodes(proxmox_node).qemu(vm_id).status.get()
+        print(f"VM {vm_id} exists. Deleting...")
+        proxmox.nodes(proxmox_node).qemu(vm_id).delete()
+        print(f"VM {vm_id} deleted.")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)